text/0000-block-let-template-helper.md

```diff
@@ -0,0 +1,249 @@
+* Start Date: 2017-12-21
+* RFC PR: https://github.com/emberjs/rfcs/pull/286
+* Ember Issue: (leave this empty)
+
+# Block `let` template helper
+
+## Summary
+
+Introduce the `let` template helper in block form.
+
+## Motivation
+
+The goal of this RFC is to introduce a `let` template helper that allows to create new bindings in templates.
+The design of this helper is similar to `with`,
+but without the conditional rendering of the block depending on the values passed into the helper.
+
+While the conditional semantics of `with` are coherent with the other built-in helpers like `each` and `if`,
+users often find this unexpected.
+The fact that only the first positional parameter of `with` controls whether the block is rendered might also add to the confusion.
+
+Taking an example from [RFC #200](https://github.com/emberjs/rfcs/pull/200),
+let's consider we have the following template:
+
+```handlebars
+Welcome back {{concat (capitalize person.firstName) ' ' (capitalize person.lastName)}}
+
+Account Details:
+First Name: {{capitalize person.firstName}}
+Last Name: {{capitalize person.lastName}}
+```
+
+Because you have to know to capitalize every time you want to display a name,
+errors might be introduced if we forget to do it when adding the name somewhere else in the template.
+Using the `let` helper, this could be done like so:
+
+```handlebars
+{{#let (capizalize person.firstName) (capitalize person.lastName)
```

Comment:
`capitalize` is misspelled here

Reply:
Thank you :)

Reply:
No worries. There's actually another misspelling further down in the document too on L125.